### PR TITLE
cephfs: remove nodeplugin RBAC for cephfs

### PR DESCRIPTION
**Pull request by humblec**
The cephfs nodeplugin RBAC is not required and this commit
remove the same.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Mergifyio rebase

**Comment by humblec** [comment]
@Madhu-1  ptal..  thanks,

**Comment by humblec** [comment]
@Mergifyio rebase

File: deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml [hunk 1]
```diff
@@ -9,9 +9,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: cephfs-csi-nodeplugin
 rules:
-  - apiGroups: [""]
```
Comment (Madhu-1) [inline deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml:12]: Why keep empty ClusterRole without any rules?

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
Need to take care in below places

* Need to remove from helm chart
* Doc update is required.
* Remove it from nfs plugin also.

**Comment by humblec** [comment]
> Need to take care in below places
> 
>     * Need to remove from helm chart
> 
>     * Doc update is required.
> 
>     * Remove it from nfs plugin also.

This was kept for CephFS specifically, however I can update the NFS side too in the same PR.

**Comment by humblec** [comment]
@Madhu-1 nfs plugin has not addressed here and will send it in a different PR.. rest of the comments are addressed. PTAL. Thanks!

File: deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml (hunk 1 above)
Comment (humblec) [reply deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml:12]: removed it

File: charts/ceph-csi-cephfs/templates/nodeplugin-clusterrole.yaml [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-metadata:
-  name: {{ include "ceph-csi-cephfs.nodeplugin.fullname" . }}
-  labels:
-    app: {{ include "ceph-csi-cephfs.name" . }}
-    chart: {{ include "ceph-csi-cephfs.chart" . }}
-    component: {{ .Values.nodeplugin.name }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-rules:
-  - apiGroups: [""]
```
Comment (Madhu-1) [inline charts/ceph-csi-cephfs/templates/nodeplugin-clusterrole.yaml:14]: clusterrole binding still needs to be removed?

Comment (humblec) [reply charts/ceph-csi-cephfs/templates/nodeplugin-clusterrole.yaml:14]: oh.. yeah.. removed it @Madhu-1

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Pull request merged**